saleor/static/dashboard-next/customers/views/CustomerAddresses.tsx

```diff
@@ -140,140 +141,99 @@ const CustomerAddresses: React.FC<CustomerAddressesProps> = ({ id }) => {
-                                    errors={maybe(
-                                      () =>
-                                        updateCustomerAddressOpts.data
-                                          .addressUpdate.errors,
-                                      []
-                                    )}
-                                    open={!!match}
-                                    variant="edit"
-                                    onClose={closeModal}
-                                    onConfirm={formData =>
-                                      updateCustomerAddress({
-                                        variables: {
-                                          id: decodeURIComponent(
-                                            match.params.addressId
-                                          ),
-                                          input: {
-                                            ...formData,
-                                            country: formData.country.value
-                                          }
-                                        }
-                                      })
+                                  }
+                                })
+                              }
+                            />
+                            <CustomerAddressDialog
+                              address={maybe(() =>
+                                customerData.data.user.addresses.find(
+                                  addr => addr.id === params.id
+                                )
+                              )}
+                              confirmButtonState={updateAddressTransitionState}
+                              countries={[]}
+                              errors={maybe(
+                                () =>
+                                  updateCustomerAddressOpts.data.addressUpdate
+                                    .errors,
+                                []
+                              )}
+                              open={params.action === "edit"}
+                              variant="edit"
+                              onClose={closeModal}
+                              onConfirm={formData =>
+                                updateCustomerAddress({
+                                  variables: {
+                                    id: params.id,
+                                    input: {
+                                      ...formData,
+                                      country: formData.country.value
                                     }
-                                  />
+                                  }
+                                })
+                              }
+                            />
+                            <ActionDialog
+                              open={params.action === "remove"}
+                              variant="delete"
+                              title={i18n.t("Remove Address")}
```

Comment:
Just wondering - wont be easier to translate `title` inside `ActionDialog`? 
I'm assuming there won't be situations when u don't intend to translate it.

Reply:
Literals, man. Unfortunately.